scvi/inference/posterior.py

```diff
@@ -224,62 +224,103 @@ def sample_scale_from_batch(self, n_samples, batchid=None, selection=None):
     def differential_expression_score(self, idx1, idx2, batchid1=None, batchid2=None,
                                       genes=None, n_samples=None, M_permutation=None, all_stats=True,
                                       sample_pairs=True):
-        if n_samples is None:
-            n_samples = 5000
-        if M_permutation is None:
-            M_permutation = 10000
+        """
+        Computes gene specfic Bayes factors using masks idx1 and idx2
+
+        To that purpose we sample the Posterior in the following way:
+            1. The posterior is sampled n_samples times for each subpopulation
+            2. For computation efficiency (posterior sampling is quite expensive), instead of
+            comparing element-wise the obtained samples, we can permute posterior samples.
+            Remember that computing the Bayes Factor requires sampling
+            q(z_A | x_A) and q(z_B | x_B)
+
+        :param idx1: MASK of subpopulation cells 1
+        :param idx2: MASK of subpopulation cells 2
+        :param batchid1: batch ids of cells 1
+        :param batchid2: batch ids of cells 2
+        :param genes: Names of genes for which Bayes factors will be computed
+        :param n_samples: Number of times the posterior will be sampled for each pop
+        :param M_permutation: Number of times we will "mix" posterior samples in step 2
```

Comment:
can we write something more precise ? I'm open to changing the name M_permutation. Also, M_permutation should follow sample_pairs ? Ideally first sample_pairs and then M_permutation since the latter is useful only provided that the former is useful only if the latter is True. for example, could we call M_permutation n_pairs ?